server/src/main/java/org/apache/iotdb/db/query/dataset/DeviceIterateDataSet.java

```diff
@@ -80,45 +77,41 @@
   private int[] currentColumnMapRelation;
```

Comment:
how about renaming this class to AlignByDeviceDataset

Reply:
It's nice. I will rename it.